packages/frontend/gulpfile.js

```diff
@@ -46,7 +46,7 @@ function watchStyles() {
 }
 
 function copyStatic() {
-  return gulp.src('src/static/**/*').pipe(gulp.dest('build'))
+  return gulp.src(['src/static/**/*']).pipe(gulp.dest('build'))
```

Comment:
why change this?

Reply:
I wanted to close this task and then move to the tsconfig changes